nixos/modules/services/misc/blenderfarm.nix

```diff
@@ -0,0 +1,141 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  cfg = config.services.blenderfarm;
+  json = pkgs.formats.json { };
+  configFile = json.generate "ServerSettings" (defaultConfig // cfg.ServerConfig);
+  defaultConfig = {
+    Port = 15000;
+    BroadcastPort = 16342;
+    BypassScriptUpdate = false;
+    BasicSecurityPassword = null;
+  };
+in
+{
+  meta.maintainers = with lib.maintainers; [ gador ];
+
+  options.services.blenderfarm = with lib.types; {
+    enable = lib.mkEnableOption "Blenderfarm, a render farm management software for Blender.";
+    package = lib.mkPackageOption pkgs "blenderfarm" { };
+    openFirewall = lib.mkEnableOption "Allow blenderfarm network access through the firewall.";
+
+    user = lib.mkOption {
+      description = "User under which blenderfarm runs.";
+      default = "blenderfarm";
+      type = str;
+    };
+
+    group = lib.mkOption {
+      description = "Group under which blenderfarm runs.";
+      default = "blenderfarm";
+      type = str;
+    };
+
+    BasicSecurityPasswordFile = lib.mkOption {
```

Comment:
If it's a native Nix option, it should be camelCase'd — same as below

Reply:
Isn't this camelCase'd?

Reply:
No, that's PascalCase 😅

Reply:
ah, gotcha 👍